NuGetCleaner/Program.cs

```diff
@@ -0,0 +1,166 @@
+            }
+        }
+
+        public static int CheckDisableLastAccess()
+        {
+            System.Diagnostics.Process process = new System.Diagnostics.Process();
+            System.Diagnostics.ProcessStartInfo startInfo = new System.Diagnostics.ProcessStartInfo();
+            startInfo.UseShellExecute = false;
+            startInfo.RedirectStandardOutput = true;
+            startInfo.FileName = "CMD.exe";
+            startInfo.Arguments = "/c fsutil behavior query disablelastaccess";
+            process.StartInfo = startInfo;
+            process.Start();
+            string output = process.StandardOutput.ReadToEnd();
+            process.WaitForExit();
+
+            string[] outputArray = output.Split(" ");
+            int setting = Convert.ToInt32(outputArray[2]);
+
+            return setting;
+        }
+
+        public static void SearchAndDestroy(string Path, int Days)
+        {
+            try
+            {
+                Console.WriteLine("Deleted Packages: ");
+
+                foreach (string pkg in Directory.GetDirectories(Path))
+                {
+                    foreach (string pkgVersion in Directory.GetDirectories(pkg))
+                    {
+                        var dirAge = DateTime.Now - RecursiveFindLAT(pkgVersion, DateTime.MinValue);
+
+                        if (dirAge.TotalDays >= Days)
+                        {
+                            Console.WriteLine(pkgVersion);
+                            RecursiveDelete(pkgVersion);
+
+                            if (Directory.GetDirectories(pkg).Length == 0)
+                            {
+                                Directory.Delete(pkg);
+                            }
+                        }
+                    }
+                }
+            }
+            catch (System.Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+        }
+
+        public static void SearchAndPrint(string Path, int Days)
+        {
+            try
+            {
+                Console.WriteLine("DRY RUN (NOTHING IS DELETED)");
+
+                Console.WriteLine("Would-Be Deleted Packages: \n");
+
+                foreach (string pkg in Directory.GetDirectories(Path))
+                {
+                    foreach (string pkgVersion in Directory.GetDirectories(pkg))
+                    {
+
+                        var dirAge = DateTime.Now - RecursiveFindLAT(pkgVersion, DateTime.MinValue);
+
+                        if (dirAge.TotalDays >= Days)
+                        {
+                            Console.WriteLine(pkgVersion.Substring(Path.Length + 1) + " --- Last Access: " + RecursiveFindLAT(pkgVersion, DateTime.MinValue));
+                        }
+                    }
+                }
+            }
+            catch (System.Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+        }
+
+        public static void RecursiveDelete(string dir)
```

Comment:
This should do it, right?
https://docs.microsoft.com/en-us/dotnet/api/system.io.directory.delete?view=netframework-4.8#System_IO_Directory_Delete_System_String_System_Boolean_